README.md

```
@@ -8,30 +8,138 @@ A zero-config CLI and MCP Server that compiles your project's architectural rule
+```typescript
+// totem.config.ts
+partitions: {
+ core: ['packages/core/'],
+ cli: ['packages/cli/'],
+ mcp: ['packages/mcp/'],
+},
+```
+
+Agents pass the partition name when searching:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)